spec/requests/assessments_controller_spec.rb

```diff
@@ -96,7 +96,7 @@
         let(:decorator) { instance_double Decorators::V5::AssessmentDecorator }
 
         it "calls the required services and uses the V5 decorator" do
-          allow(Decorators::V5::AssessmentDecorator).to receive(:new).with(assessment).and_return(decorator)
+          allow(Decorators::V5::AssessmentDecorator).to receive(:new).with(assessment, nil).and_return(decorator)
```

Comment:
think this is the cause of the safe navigation operators above - maybe better to replace with an empty CalculationOutput rather than nil